ndohyep/registration/register.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load static core_tags %}
+{% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
```

Comment:
@miltontony just told me that Wagtail makes it available in the template context.